### DISCUSSION TITLE
Market Analysis

### DISCUSSION THREAD

**Discussion by charlieforward9** [charlieforward9/Curvify#1] [General]
There is currently software in the market that simulates the outcome of plastic surgery procedures on the face and body in executable (cross-platform app), and browser options. 

However, it does not look like it uses generative AI. In fact, there is no mention of AI with these products. With the current AI boom, we can position ourselves competitively by marketing ourselves as an exclusive AI solution. Furthermore, with a generative foundation, we should be able to tackle far more edge cases than a non-generative, 'hard-coded' approach.

Even if we are not the FIRST to develop this solution, we have the opportunity to learn from our competitors, develop a more robust product, and tap into an attractive $26.6B industry.

**Comment by charlieforward9** [comment]
# Problem
Women want to know what surgical breast alterations will look like before committing to an expensive and permanent operation. Currently, the surgeon provides their client with a surgical simulation to give patients a realistic expectation of the outcome, but there are many limitations to the current approach: 

### What currently exists? 
* Web-based, non-secure software (SaaS)
* Expensive, space-consuming imaging booths (Hardware)

### How is it being delivered to the user?
* Limited POV, static images

# Solution
* Mobile, physician-licensed, generative-based SaaS
* Real-time image generation for a "real-life look" at the alteration. (30-60 generations per second)

@ManuelCortes23 How feasible is that real-time image generation? Going for a Snapchat filter feel but using Generative AI to make the image alteration. That alone would be a huge competitive advantage over static images.

**Reply by ManuelCortes23** [reply]
Real time generation with AI models is pretty cutting-edge research, and the computational resources needed for it potentially make it unfeasible as an initial scope.

**Reply by charlieforward9** [reply]
It can be trained using images of celebrities that have gotten procedures. Before/Afters

**Comment by charlieforward9** [comment]
# Competitors 
### Companies specialized in providing surgical simulation technologies: 

**Crisalix**
Offers 3D and 4D imaging for aesthetic and reconstructive surgery, allowing both doctors and patients to visualize potential results.

**Touch Surgery**
Provides a surgical simulation platform aimed at training and planning, covering a wide range of surgical procedures.

**3D Systems**
Offers a range of healthcare solutions including surgical simulations for training and preoperative planning.

**Mentice**
Specializes in endovascular simulations, offering realistic training environments for complex surgeries.

**Surgical Science**
Provides training simulations for laparoscopic procedures, aimed at skill development for practicing surgeons.

**Simbionix**
Offers a range of simulation solutions for multiple types of medical procedures, including laparoscopic, endovascular, and more.

**CAE Healthcare**
Provides training simulations for a variety of medical procedures and practices, though their focus is broader than just surgical simulation.

**VirtaMed**
Specializes in simulation for orthopedic and laparoscopic procedures, providing highly realistic training environments.

**Intuitive Surgical**
Known primarily for their da Vinci surgical systems, they also offer simulation modules for robotic surgery training.

**Medtronic**
Provides simulation and training technologies in addition to their broad range of medical devices.

**Comment by charlieforward9** [comment]
# Limitations

Accuracy: No simulation can perfectly predict post-operative results. Even with advanced AI and 3D modeling, the human body's response to surgery is subject to numerous variables that can't be fully accounted for.

Cost: High-quality simulation software can be expensive, which may limit access for smaller medical practices or educational institutions.

Hardware Requirements: Advanced simulation programs may require specialized hardware, further increasing costs and potentially creating compatibility issues.

Learning Curve: Utilizing complex simulation software effectively often requires specialized training, adding an additional time investment for medical professionals who are already extremely busy.

Patient Misunderstanding: Even though simulations aim to provide a realistic preview, there's a risk that patients may misunderstand the limitations of these previews, leading to unrealistic expectations.

Ethical Concerns: There may be issues surrounding informed consent if patients misconstrue simulations as guarantees of surgical outcomes.

Computational Limitations: Rendering realistic and high-definition models can be computationally intensive, requiring more time or more powerful hardware to produce results.

Software Limitations: Some simulations may lack the ability to accurately model complex interactions between tissues, blood flow, or other physiological factors that could influence surgical outcomes.

Standardization: Lack of standard protocols for performing and interpreting simulations can lead to inconsistent use and results.

Interdisciplinary Communication: In complex surgeries requiring the expertise of surgeons from different specialties, there may be limitations in how effectively a simulation can facilitate interdisciplinary planning and communication.

Data Privacy and Security: As with any digital health technology, there are concerns about how patient data is stored, accessed, and protected.

Regulatory Approval: Depending on the jurisdiction, medical software used for diagnosis or treatment planning may need to undergo rigorous approval processes, which can limit quick adoption of newer, perhaps better, technologies.

Scalability: For hospitals or practices with a high volume of patients, the time required to create individual simulations could become a bottleneck in the planning process.

Tissue Properties: Current simulations may not adequately account for the unique properties of different tissues, which could be especially relevant in fields like plastic surgery, where esthetic outcomes are paramount.

_Thanks ChatGPT_

**Reply by conorgiven** [reply]
In response to 'Accuracy'- We could use published data on innate (nonspecific and naturally occurring to any foreign object) immune response to specific materials as a model input. Without data analysis, it can be predicted that this should result in slightly enlarged breast due to scar tissue/immune response in area directly around implant.

**Reply by charlieforward9** [reply]
Sounds like grantable research right here

**Comment by conorgiven** [comment]
Material Type

Currently, saline and silicone gel implants dominate the breast augmentation market. In addition, physicians currently use high-resolution ultrasound to evaluate post-operative efficacy; however, there in no mention of a control comparison and it appears they are only looking for rippling which is external appearance of folds/creases in the material implant itself. It appears that, as mentioned in one of the sections above, there is no AI use and instead static images are compared side-by-side, potentially leading to error in post-op analysis as well. 

Rippling and Implant Volume- It appears that there is a statistically significant correlation between evident rippling and lower implant volume as well as lower body mass indices (P<0.05). This data is from a prospective study with n=237 so we may need to find a larger data pool later on. However, these could be interesting input variables for our model. We will need to use data analysis to train the model to predict rippling probability based off a variety of different input variables and how prevalent/obvious such effect would be. Current data also shows a positive correlation between rippling and time. It may be of use to offer the patient a predictive view at different time points (not just time post-op).

Material- it appears that silicone gel implants offer a greater aesthetic advantage over saline implants (different material properties and thus appear different following surgery). It is of ethical note that textured implants offer an increased propensity for the development of anaplastic large-cell lymphoma (cancer) compared to smooth implants. We should therefore focus our model training on smooth saline versus silicone gel implants as they are predominantly used by surgeons. Another variable of note is the cross-linking density of the silicone gel. Greater cross-linking is associated with greater material firmness (different view post-op) and cause less rippling. Silicone implants, if they leak, are prone to granulation (inflammatory response), but we can probably ignore this since it is a worst-case scenario. Saline implants are typically more round and firm to the touch while silicone gel implants better mimic true breast tissue. 

Patient-Specific Variables- The primary focus on breast augmentation in women involves proportionality. Therefore, to provide a realistic predictive model, full-body or at the very least torso scans should be our focus. Taller/great BMI women will require greater volume to produce equal proportionality. In addition, proportionality/appearance can be impacted by patient-specific factors such as thorax shape, pectus excavatum, etc.. Are we able to train a model to recognize this during uploaded scans?

Further Market Research- This quote is from a review published in 2014, however, further illustrates the demand for our product:

Preoperative sizing is another method that is more subjective in its approach. It consists of placing sample implants in a bra to preview a range of possible results. The surgeon first determines a size range suggested by height, weight, and body habitus that is also mindful of breast anatomy restrictions. This process shares ownership of the final decision between the patient and the surgeon. It has been shown to minimize requests for size-change surgery.

Sources:
1) https://journals.lww.com/prsgo/fulltext/2020/06000/prospective_study_of_saline_versus_silicone_gel.14.aspx
2) DOI:  10.1097/PRS.0000000000000033
3) https://advancedcosmeticsurgery-sc.com/silicone-vs-saline-implants/#:~:text=Saline%20implants%20have%20a%20distinctly,and%20more%20malleable%20in%20texture.
4) https://watermark.silverchair.com/asj_39_3_264.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAA3owggN2BgkqhkiG9w0BBwagggNnMIIDYwIBADCCA1wGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMdLt3BhQG1hwZKSaSAgEQgIIDLa0vnKhWUCpbwqACJwoZew3qdEgTA9D-G_iAEjZB_86k6yMB8h10KpOW1svcBJ9f_2J_6-Ic21AvGwy7JOd-vOZ2GgaKGQkS3TYl_aUaVF3-cKHxBkxoooZ4xqHdJEFE4pBqzDAt1k5T0XCQfCOmo_58G5-u3hfjkNIDO-Q982c_encA8lCQRwJFI1a4soHc_9j1eDlrfiKynUI9nQ9OKYJi4wYp_CK3wmMhQy309rQtCUQmJvnmYvJP318gd_6zVcZ4hEF-xAPaI1YSDuj0Zq41L0IYkBfvWbymXAZDNiJP8LbniGnDmU4D2-NBG4oU8dmsdXukuKKIDYbmx289l-1dYo4uUNqb2qDTQPcf8PqNcgb7SE2hhz91CdayclV27iabWdltuGhz9lAmBa-P_AFdNoKlqWbrgYQ-ey9a3UM0hExCR-mIV5lHKMQIBORoj2C4t3NtZMch3WAePOvxRFAAR5QH50EzFG2_l6ERA1bFgVBQzrYkXzwALvHM589ET8DTeEH-VQrjfnV-bEbVBFbb8ZemzHRUfMYCihMgGxEKzNiSrqsx3xXZvmlf_i0A8FiUQNmrnNwPem_CUkzr1Xyc7Z3Rm9DVNLcivxfwkvoZcNYzr02chys6iXQnWBSFEq_0NsLMxTo_86b906Ll79bSqGEEtbyr3pJaNc2ENphhsBge0nKFvAWtZO8MdO3caPn624lxwe9oiDxeS6_Ac-xBfABgtRTkMMBhvJCaMCRKtkR_qh0k3dz9CSio65U489w2zq5TKVQ_DAQ6CIMr-83Oyy1nocQ2wyqupuNYW9j-7BiMvGp76axISAeeOPxANc-V8swx2Y84cTygq5sPPV2HSLkOmBcK9IHRMgDZBcE3IreevHeGIILOsmEA5uqMIjkBnHdnVaq81QC7_6kpmhTq_dqaKHk7ieaauQ4xtu3gYz1_Wz-TuCxsiUR97H0Ei5pw1AKbLxGweK4mRjGttYJe9FoyxVI5JU_l1BnyeYsBoTYN0i-3O0VH1um6o6LbOn1OT5cxzXtMvNjTlOlAG4kCjNqWuNvGCQ8EdNqIgYQkh0Ojoy9w7ehN9eSOpQ

**Reply by charlieforward9** [reply]
Business might have to start booming
